src/daemon/rpmostreed-transaction-livefs.c

```diff
@@ -659,8 +659,9 @@ livefs_transaction_execute_inner (LiveFsTransaction *self,
         return FALSE;
     }
 
-  /* XXX: right now we don't have a good solution for this */
-  rpmostree_output_message ("WARNING: changes to /etc are not currently preserved!");
+  /* XXX: right now we don't have a good solution for this:
+   * https://github.com/projectatomic/rpm-ostree/issues/40 */
+  rpmostree_output_message ("WARNING: changes to /etc will not appear in the pending deployment (next reboot)");
```

Comment:
still seems ambiguous `WARNING: new changes to /etc will not appear in the pending deployment (i.e. will be lost on next reboot)`